bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/TestHelper.java

```diff
@@ -49,7 +50,13 @@ public static void cleanUpStorageFolder(String gsPath) throws IOException {
     for (GcsPath path : gcsUtil.expand(GcsPath.fromUri(gsPath + "*"))) {
       pathsToRemove.add(path.toString());
     }
-    pathsToRemove.add(gsPath);
+    try {
+      if (gcsUtil.getObject(GcsPath.fromUri(gsPath)) != null) {
```

Comment:
how can the path exist?

Reply:
This path is probably created when Import/Export tools are working, not by the tests.
The tests do something like:

```
  BeforeClass:
    gcsPath = TestHelper.getTestProperty("google.dataflow.gcsPath");
    gcsWorkDir = gcsPath + "/" + UUID.randomUUID().toString();
    stagingLocation = gcsWorkDir + "/staging";
    tempLocation = gcsWorkDir + "/temp";


  During testing:
    DataflowPipelineOptions options = PipelineOptionsFactory.as(DataflowPipelineOptions.class);
    options.setStagingLocation(stagingLocation);
    options.setTempLocation(tempLocation);


  AfterClass:
    TestHelper.cleanUpStorageFolder(stagingLocation);
    TestHelper.cleanUpStorageFolder(tempLocation);
    TestHelper.cleanUpStorageFolder(gcsWorkDir);
```

Without removing the path under question the bucket is not properly cleaned up:
```
#> gsutil ls gs://bucket_for_bt_test/
gs://bucket_for_bt_test/546d262a-8a5b-41c9-b0ad-e12415576238/
#> gsutil ls gs://bucket_for_bt_test/546d262a-8a5b-41c9-b0ad-e12415576238/
gs://bucket_for_bt_test/546d262a-8a5b-41c9-b0ad-e12415576238/temp/
```

Ideally, it should be enough to call `TestHelper.cleanUpStorageFolder(gcsWorkDir)`, but I didn't find how to perform
`gsutil rm -rf` with currently used `org.apache.beam.sdk.extensions.gcp.util.GcsUtil`.
So, removing the extra path helps to keep the bucket clear from the garbage.